zip-0230.rst

```diff
@@ -4,11 +4,347 @@
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``32``                              |``ik``                    |``byte[32]``                            |The issuance validating key of the issuer, used to validate the signature. |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64``                              |``issueAuthSig``          |``byte[64]``                            |The signature of the transaction SIGHASH, signed by the issuer,            |
+|                                    |                          |                                        |validated as in Issuance Authorization Signature Scheme [#zip-0227]_.      |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+
+
+* The fields ``valueBalanceSapling`` and ``bindingSigSapling`` are present if and only if
+  :math:`\mathtt{nSpendsSapling} + \mathtt{nOutputsSapling} > 0`. If ``valueBalanceSapling``
+  is not present, then :math:`\mathsf{v^{balanceSapling}}`` is defined to be 0.
+
+* The field ``anchorSapling`` is present if and only if :math:`\mathtt{nSpendsSapling} > 0`.
+
+* The fields ``flagsOrchard``, ``valueBalanceOrchard``, ``anchorOrchard``,
+  ``sizeProofsOrchardZSA``, ``proofsOrchardZSA``, and ``bindingSigOrchard`` are present if and
+  only if :math:`\mathtt{nActionsOrchard} > 0`. If ``valueBalanceOrchard`` is not present,
+  then :math:`\mathsf{v^{balanceOrchard}}` is defined to be 0.
+
+* The elements of ``vSpendProofsSapling`` and ``vSpendAuthSigsSapling`` have a 1:1
+  correspondence to the elements of ``vSpendsSapling`` and MUST be ordered such that the
+  proof or signature at a given index corresponds to the ``SpendDescriptionV6`` at the
+  same index.
+
+* The elements of ``vOutputProofsSapling`` have a 1:1 correspondence to the elements of
+  ``vOutputsSapling`` and MUST be ordered such that the proof at a given index corresponds
+  to the ``OutputDescriptionV6`` at the same index.
+
+* The proofs aggregated in ``proofsOrchardZSA``, and the elements of
+  ``vSpendAuthSigsOrchard``, each have a 1:1 correspondence to the elements of
+  ``vActionsOrchard`` and MUST be ordered such that the proof or signature at a given
+  index corresponds to the ``OrchardZsaAction`` at the same index.
+
+* For coinbase transactions, the ``enableSpendsOrchard`` and ``enableZSAs`` bits MUST be set to ``0``.
+
+The encodings of ``tx_in``, and ``tx_out`` are as in a version 4 transaction (i.e.
+unchanged from Canopy). The encodings of ``SpendDescriptionV6``, ``OutputDescriptionV6``
```

Comment:
It's a bit arbitrary to say Canopy here; NU5 is the last upgrade, and I think they are unchanged from launch. But this is non-blocking.

Reply:
Fixed this